### PR TITLE
Quid solved

### PR DESCRIPTION
**Pull request by Collins-Webdev**
### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch
- [x] I have tested my code with the `pytest` command as directed in the [testing section of the contributing guide](https://github.com/scribe-org/Scribe-Data/blob/main/CONTRIBUTING.md#testing)

---

### Related issue

- #ISSUE_NUMBER

### CODE REVIEW

**Comment by Collins-Webdev** [comment]
Hello @catreedle 👋🏼, 
Thank you for your feedback 🙏🏽.
I've made the following changes based on your comments:

1. Removed the assumption of a 'languages' key in the language_metadata dictionary. The code now iterates directly over the language_metadata items.
2. Added handling for sub-languages. The code now checks for the presence of 'sub_languages' before processing the main language qid.
3. Improved the warning messages to differentiate between missing qids for main languages and sub-languages.

Please review the changes and let me know if any further modifications are needed.

File: src/scribe_data/cli/cli_utils.py [hunk 1]
```diff
@@ -27,8 +27,6 @@
 
 from scribe_data.utils import DEFAULT_JSON_EXPORT_DIR
 
-# MARK: CLI Variables
```
Comment (andrewtavis) [inline src/scribe_data/cli/cli_utils.py:30]: Please don't remove marks from the code in the future @Collins-Webdev as these are meant to make the files more manageable to navigate.

**Comment by andrewtavis** [comment]
50d4c30 further added back in some single quotes that were removed and were causing the tests to fail :)

**Review by andrewtavis — APPROVED** [review]
Thanks for the changes here, @Collins-Webdev! And also appreciate the review, @catreedle! Made mine much easier 😊 Would be great to get continued support on checking PRs!

**Pull request merged**